wgpu-types/src/lib.rs

```diff
@@ -764,6 +764,22 @@ impl BlendComponent {
         operation: BlendOperation::Add,
     };
 
+    /// Color blend mode that does standard alpha blending with non-premultiplied alpha.
+    /// The alpha blend mode often used is REPLACE.
+    pub const COLOR_BLEND: Self = BlendComponent {
+        src_factor: BlendFactor::SrcAlpha,
+        dst_factor: BlendFactor::OneMinusSrcAlpha,
+        operation: BlendOperation::Add,
+    };
+
+    /// Color blend mode that does standard alpha blending with premultiplied alpha.
+    /// The alpha blend mode often used is REPLACE.
+    pub const COLOR_BLEND_PREMULTIPLIED_ALPHA: Self = BlendComponent {
```

Comment:
if the first one is `ALPHA_BLEND`, then this one would be `PREMULTIPLIED_ALPHA_BLEND`